Exiled.API/Features/Attributes/CustomItemAttribute.cs

```diff
@@ -7,7 +7,6 @@
 
 namespace Exiled.API.Features.Attributes
 {
-#pragma warning disable 1584
```

Comment:
Do not remove them or @joker-119 will cry.

Reply:
I'm ready to make a sacrifice

Reply:
And I'm ready to not approve nor merge it unless you solve what I just pointed out.

Reply:
And I only need to revert warning 1584?

Reply:
Okay, i'll revert it tomorrow

Reply:
mmk